Packs/CadoResponse/Integrations/CadoResponse/CadoResponse.py

```diff
@@ -0,0 +1,493 @@
+
+        payload: Dict[str, Any] = {
+            'bucket': bucket,
+            'compress': compress,
+            'include_disks': include_disks,
+            'include_hash': include_hash,
+            'include_logs': include_logs,
+            'include_screenshot': include_screenshot,
+            'instance_id': instance_id,
+            'region': region
+        }
+
+        return self._http_request(
+            method='POST',
+            url_suffix=f'/projects/{project_id}/imports/ec2',
+            json_data=payload
+        )
+
+    def trigger_bucket_acquisition(self, project_id: Optional[int], bucket: Optional[str],
+                                   file_name: Optional[str]) -> Dict[str, Any]:
+        ''' Calls the POST /api/v2/projects/{id}/imports/s3 endpoint to
+            trigger an acquisition of a given bucket or file
+
+            :param Optional[int] project_id: The ID of the project you wish to attach the acquisition to
+            :param Optional[str] bucket: The S3 bucket name containing the file
+            :param str file_name: The name of the file to process
+
+            :return JSON response from /projects/{id}/imports/ec2 endpoint
+            :rtype Dict[str, Any]
+        '''
+
+        if not project_id:
+            project_id = demisto.params().get('CadoResponse_DefaultProject', 1)
+
+        if not bucket:
+            bucket = demisto.params().get('CadoResponse_DefaultBucket', 'cado-default-bucket')
+
+        payload: Dict[str, Any] = {
+            'bucket': bucket,
+            'file_name': file_name
+        }
+
+        return self._http_request(
+            method='POST',
+            url_suffix=f'/projects/{project_id}/imports/s3',
+            json_data=payload
+        )
+
+
+''' Command Line Handlers '''
+
+
+def test_module(client: Client) -> str:
+    ''' Command handler for !test-module '''
+
+    result: Dict[str, Any] = client.heartbeat()
+    status: Optional[str] = result['status']
+
+    if status is not None and status == 'Running':
+        return 'ok'
+
+    return 'Cado Response is not running'
+
+
+def create_project_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    ''' Command handler for cado-create-project '''
+
+    unix_timestamp: str = str(int(time.time()))
+    project_name: str = args.get('project_name', unix_timestamp)
+    project_description: Optional[str] = args.get('project_description', None)
+    result: Dict[str, Any] = client.create_project(project_name, project_description)
+
+    return CommandResults(
+        outputs_prefix='CadoResponse.Project',
+        outputs_key_field='id',
+        outputs=result
+    )
+
+
+def list_project_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    ''' Command handler for cado-list-project '''
+
+    project_id: Optional[int] = args.get('project_id', None)
+    limit: int = int(args.get('limit', 50))
+
+    if project_id:
+        result: Any = client.get_project(project_id)
```

Comment:
in case there's no project with the given id, an 404 error would be returned, in such case we'd want to return a readable output that states so, and not return an error.
for list of projects, if i does not return an empty list (return an error), then it should be handled the same way.